infrastructure/lambda/template.yaml

```diff
@@ -190,6 +202,44 @@ Mappings:
       production: "https://api.os.uk/search/places/v1/postcode"
 
 Resources:
+  #############################
```

Comment:
Do we still need all this mock infrastructure see https://gds.slack.com/archives/C075PRX6SBF/p1717402909752899?thread_ts=1717071228.520419&cid=C075PRX6SBF I think we can just use the really TxMA one